web_development_101/git_basics/introduction_to_git.md

```diff
@@ -42,6 +42,8 @@ By the end of this lesson, you should be able to do the following:
 This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
 
 * [Git and Github in plain English](https://blog.red-badger.com/blog/2016/11/29/gitgithub-in-plain-english)
+* Learn Git in 30 Minutes - from [codeSTACKr]
```

Comment:
Please make sure the markup is correct, the tag and the link has to be on the same line